### ISSUE TITLE
Stat logging, actual cookie gain per hour or other timeframe

### ISSUE THREAD

**Issue by TheDutchman88** [greenc/CookieMaster#40]
What i would love to see is that the actual cookie gain per hour is tracked. So cookies gained in the last hour and your average cookies per hour based on the last x number of hours tracked. (so not the time since last reset but the time the game is running)

Now calculations on remaining time is purely based on CpS, which changes with every frenzy and doesnt include the minutes/hours worth of gain for every golden cookie, frenzy's etc

So when you need for example 1 quintillion cookies, even though the timer says 18 hours, the amount is met after playing a mere 30 minues, so if the stat logger gives me an average gain of 2 quintillion cookies per hour i know that i only need to play for 30 minutes instead of 18 hours.

This becomes especially interesting end game when the cookies needed are in the quint/sextillions of cookies

_EDIT_
another way that increases the acuracy atleast somewhat is to use the current stat logging where it tracks effective CpS and based on those numbers adding another line giving the average CpS, and then using that average CpS for time calculations instead of the effective CpS. this stil does not include the gain trough lucky golden cookies though.

**Comment by ghost** [comment]
Another way would be: before an update is bought calculate

``` javascript
(cookiesSinceLastUpgrade / secondsSinceLastUpgrade) / baseCPS
```

This multiplier, averaged of the last hour (Edit: weighted by the secondsSinceLastUpdate), should give a fairly good estimation for long timers.
It would be nice to see something like estimated remaining time based on this for times which are longer than ~3 mins.

**Comment by RuiApostolo** [comment]
while we're on the logging subject, is it possible to make it so that in the .csv file, the time column comes in seconds, rather than in "##h##m##s" ?

**Comment by greenc** [comment]
Yes, this would be possible. I'm currently working on other features at the moment, but I do want to revisit the stat logging and improve upon it, as currently it's not very robust and a bit thin on actual functionality.

**Comment by Casshern1324** [comment]
Also, I would like to ask for a way to keep the old way Time Remaining was calculated (Only using current CpS). As I find it weird for a building to stay at 5 minutes 8 seconds for longer than 10 seconds...
